README.md

```diff
@@ -127,6 +127,17 @@ See http://semver.org/.
 
 Please check [CONTRIBUTING](CONTRIBUTING.md) before making a contribution.
 
+## Release flow
+
+As of 2019-12, releaser is LINE's employee.
+
+Basically, versions defined in `gradle.properties`.
+
+1. Before update, update `gradle.properties` to remove `-SNAPSHOT`. e.g (`version=3.2.0`)
+2. Commit `gradle.properties` and tag version name. example: `git add -u && git commit -m 'Release 3.2.0' && git tag '3.2.0' && git push`
+3. Proceeding release action (upload central etc.)
```

Comment:
3. Upload the artifacts to the staging repository
4. Close and release the staging repository at https://oss.sonatype.org/